ui/browser/views/navbar/navbar.jsx

```diff
@@ -20,8 +20,8 @@ import Location from './location.jsx';
 function NavBar(props) {
   const {
     navBack, navForward, navRefresh, page, pages, openMenu, ipcRenderer,
```

Comment:
Comment, not issue: this list of props seems untenable over time.  Is this really how React Apps are built?